src/javascript/SelectorTypes/SelectorTypes.js

```diff
@@ -19,7 +19,15 @@ const adaptDateProperty = (field, property) => {
 export const registerSelectorTypes = ceRegistry => {
     ceRegistry.add('selectorType', 'Category', {cmp: Category, supportMultiple: true});
     ceRegistry.add('selectorType', 'Tag', {cmp: Tag, supportMultiple: true});
-    ceRegistry.add('selectorType', 'Text', {cmp: Text, supportMultiple: false});
+    ceRegistry.add('selectorType', 'Text', {
+        cmp: Text,
+        supportMultiple: false,
+        adaptValue: (field, property) => {
+            if (field.selectorOptions?.find(option => option.name === 'password')) {
+                return property.encryptedValue;
+            }
+        }
+    });
```

Comment:
Your Backend API do not provide multiple support for this property to be set, so how can it work here in case the field is multiple ? it will not.
Backend should provide a multiple support.